internal/npm_install/bulk_copy.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
missed this on the first PR. why not
```suggestion
#! /usr/bin/env bash
```
?